65black/StackOlite
# #160201358 Delete A Question

#### What does this PR do?
This PR consumes the `DELETE /api/v1/questions/:questionId/` endpoint from [this repo](https://github.com/65black/StackOverflow-lite).

#### How should this be manually tested?
 - Clone this repo and open index.html with Chrome.
 - Log in to an existing account.
 - Navigate to questions page.
 - Open any question you own.
 - You should see a small button under the content body.
 - Click it and click 'ok' on the subsequent confirm dialog. 
 - Your question will be deleted and you should be redirected to your profile page.

#### Screenshots
![delete](https://user-images.githubusercontent.com/36507972/45931066-1d0da800-bf60-11e8-8605-12ac213f1df3.JPG)

#### What are the relevant pivotal tracker stories?
#160201358
